src/languages/javascript/index.ts

```diff
@@ -199,62 +209,82 @@ const javascriptLanguage: Language = {
         int_to_bool: (x) => implicitConversion("int_to_bool", x[0]),
         "contains[Table]": (x) => infix("in", x[1], x[0]),
       }),
-      mapTo((name: string, [obj, ...args]) => method(obj, name, ...args))({
+      mapMutationTo.prefix({
```

Comment:
js doesn't seem to be generating prefix mutations right now, compiling
```
$x:0..10 <- 1;
$x <- (+ $x 1):0..10;
```
produces
```js
x=1
x+=1
```

Reply:
Fixed.